### PR TITLE
Updated and renamed ETL.md to cloud_integration.md

### PR DESCRIPTION
**Pull request by lgardenhire**

### CODE REVIEW

**Comment by lgardenhire** [comment]
@benfred - This doc needs to move to the Resources folder.

File: docs/source/cloud_integration.md [hunk 1]
```diff
@@ -49,16 +49,16 @@ To run NVTabular on the cloud using AWS, do the following:
    jupyter-lab --allow-root --ip='0.0.0.0' --NotebookApp.token='<password>'
    ```
 
-## Google Cloud Platform
+## Google Cloud Platform ##
 
-The Google Cloud Platform offers [Compute Engine instances with NVIDIA GPU support](https://cloud.google.com/compute/docs/gpus). We're using a VM with 8x NVIDIA A100 GPUs and eight local SSD-NVMe devices configured as RAID 0 to demonstrate the steps below.
+The Google Cloud Platform offers [Compute Engine instances with NVIDIA GPU support](https://cloud.google.com/compute/docs/gpus). We're using a VM with 8x NVIDIA A100 GPUs and eight local SSD-NVME devices configured as RAID 0 to demonstrate the steps below.
```
Comment (karlhigley) [inline docs/source/cloud_integration.md:54]: It looks weird, but I think [`NVMe`](https://nvmexpress.org/) is actually correct here.

File: docs/source/cloud_integration.md [hunk 2]
```diff
@@ -49,16 +49,16 @@ To run NVTabular on the cloud using AWS, do the following:
...
 
 To run NVTabular on the cloud using GCP, do the following:
 
 1. Configure and create the VM.
     * **GPU**: 8xA100 (a2-highgpu-8g)
     * **Boot Disk**: Ubuntu version 18.04
-    * **Storage**: Local 8xSSD-NVMe
+    * **Storage**: Local 8xSSD-NVME
```
Comment (karlhigley) [inline docs/source/cloud_integration.md:61]: See above re: `NVMe`.

File: docs/source/cloud_integration.md [hunk 3]
```diff
@@ -96,5 +96,5 @@ To run NVTabular on the cloud using GCP, do the following:
 
 5. Run the container using the following command:
    ```
-   docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE -v /mnt/raid:/raid nvcr.io/nvidia/nvtabular:0.3 /bin/bash
+   docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host --cap-add SYS_PTRACE -v /mnt/raid:/raid nvcr.io/nvidia/nvtabular:0.4 /bin/bash
```
Comment (benfred) [inline docs/source/cloud_integration.md:99]: This isn't the containers we're going with - we should either leave on the 0.3 tag for now (which is valid) or switch over to the merlin containers for 0.4  (https://docs.google.com/document/d/1fJqK-NZnY7xIUGoPhXRMPjmwSRz--VNR0z9pb0_fptU/edit?disco=AAAALZ0SuYI&ts=60394597&usp_dm=false) . 

I think for now lets just leave at 0.3 - we will have other updates/version bumps needed for release

File: docs/source/cloud_integration.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
-ETL
-===
+Cloud Integration
```
Comment (benfred) [inline docs/source/cloud_integration.md:1]: Renaming the file will break the docs without also updating the TOC in docs/source/index.rst

File: docs/source/cloud_integration.md (hunk 1 above)
Comment (lgardenhire) [reply docs/source/cloud_integration.md:54]: @karlhigley - Ronay had called it out. You can leave it if it's correct.

File: docs/source/cloud_integration.md (hunk 4 above)
Comment (lgardenhire) [reply docs/source/cloud_integration.md:1]: @benfred - This doc should now appear under Resources. I'll remove it completely from docs/source/index.rst.

File: docs/source/cloud_integration.md (hunk 1 above)
Comment (lgardenhire) [reply docs/source/cloud_integration.md]: I just reverted all NVME references to NVMe.

File: docs/source/cloud_integration.md (hunk 3 above)
Comment (lgardenhire) [reply docs/source/cloud_integration.md]: I just added 0.3 back.

**Review by benfred — APPROVED** [review]

**Pull request merged**